package.json

```diff
@@ -12,6 +12,9 @@
       "url": "https://github.com/tryretool/react-retool/blob/main/LICENSE"
     }
   ],
+  "types": "Retool.d.ts",
+  "main": "Retool.js",
+  "type": "commonjs",
```

Comment:
the build script could be edited to rename these to `index.*`,  (although that'd have no actual effect) but without these lines, my project depending on this wouldn't build.